### PR TITLE
pre-account ping with histograms

### PR DESCRIPTION
**Pull request by badboy**
Things to note:

* The JSON definition of histograms contains more than we actually have now, but it's all optional, so it's fine
* I still marked everything of a histogram required in the parquet, because without these fields it's not valid histogram data anyway. Is that ok?
* I'm still worried that scalars are marked as `int64` only. While that is no problem _right now_, we might as well have non-integer scalars in the pre-account ping eventually.

Checklist for reviewer:

- [ ] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [ ] Trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional)

### CODE REVIEW

**Comment by badboy** [comment]
Can we get this in? @fbertsch? @whd?

File: templates/telemetry/pre-account/pre-account.4.schema.json [hunk 1]
```diff
@@ -113,13 +113,32 @@
...
-          "type": "object"
+          "type": "object",
+          "additionalProperties": {
+            "type": "object",
+            "additionalProperties": @TELEMETRY_SCALARS_1_JSON@
+          }
         },
         "histograms": {
-          "type": "object"
+          "type": "object",
```
[the remark this answers is not in the document]
Comment (badboy) [reply templates/telemetry/pre-account/pre-account.4.schema.json]: That would mean extending it when we actually would get more processes?

**Comment by badboy** [comment]
> Aside: Apologies for the super-late review :/

Don't worry. No one's currently depending on this data to be accessible, otherwise I would have pushed for it more.

**Comment by jklukas** [comment]
Closing this as no longer required. pre-account ping is deprecated in favor of account ecosystem telemetry.

**Pull request closed without merging**